var/spack/repos/builtin/packages/spykfunc/package.py

```diff
@@ -30,8 +30,9 @@ class Spykfunc(PythonPackage):
     version('0.12.2', tag='v0.12.2', submodules=True, get_full_repo=True)
 
     depends_on('cmake', type='build', when='@0.15.4:')
-    depends_on('boost', type=('build', 'link'), when='@0.15.4:')
+    depends_on('boost', type=('build', 'link'), when='@0.15.4:0.17.0')
```

Comment:
Is Boost needed up to `v0.17.0`, included? Otherwise, we should say something like `@0.15.4:0.16.9999`, what do you think?

Reply:
RIght :facepalm:

Reply:
```suggestion
    depends_on('boost', type=('build', 'link'), when='@0.15.4:0.16.999')
```